### DISCUSSION TITLE
better-sqlite3

### DISCUSSION THREAD

**Discussion by DaStormer** [sequelize/sequelize#13455] [Help & Questions]
Are there any plans of using [better-sqlite3](https://github.com/JoshuaWise/better-sqlite3) instead of sqlite3?

**Comment by ryuzaki01** [comment]
there is https://github.com/sequelize/sequelize/issues/11400, but no progress

**Reply by WikiRik** [reply]
There indeed seems to be that there is no progress yet. I also do not know how high the priority on this is, but if people are willing to work on it they can just submit a pull request.
